daemon/daemon.go

```diff
@@ -101,7 +102,11 @@ func (c *Command) canAccess(r *http.Request, user *auth.UserState) bool {
 		}
 
 		if c.PolkitOK != "" {
-			if authorized, err := polkitCheckAuthorizationForPid(pid, c.PolkitOK, nil, polkit.CheckAllowInteraction); err == nil {
+			var flags polkit.CheckFlags
+			if r.Header.Get(client.AllowInteractionHeader) == "1" {
```

Comment:
Same comment about the "true" thing. I think we also may have a helper for that but perhaps I'm confused (helper about boolean things like this).

Reply:
my suggestion here would be to have the client send `"true"`, but use `strconv.ParseBool` in the ademon to accept anything true-ish.